src/views/migrationpolicies/list/MigrationPoliciesList.tsx

```diff
@@ -28,6 +28,9 @@ const MigrationPoliciesList: FC = () => {
     isList: true,
     namespaced: false,
   });
+  const isUnprivilegedUser =
+    !loaded &&
+    loadError?.toString()?.includes('migrationpolicies.migrations.kubevirt.io is forbidden');
```

Comment:
please use access check and not trying to parse an error msg, the error can be changed without any notice.
u can find examples in the permissions hooks that we have

Reply:
Can you please provide more details about what you mean? Haven't found anything related yet.. Thanks.

Reply:
for example https://github.com/kubevirt-ui/kubevirt-plugin/blob/main/src/utils/hooks/usePermissions/usePermissions.ts

Reply:
This hook seems to be for totally a different use!

Reply:
This hook is an example of checking permissions for the user; it's exactly what is needed to know if the button should be disabled or not based on permissions.